lib/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import classnames from 'classnames';
+import { useStyles } from 'sku/treat';
+import { Theme } from 'treat/theme';
+import { Box } from '../Box/Box';
+import { useBackground } from '../Box/BackgroundContext';
+import { UseBoxStylesProps } from '../Box/useBoxStyles';
+import * as styleRefs from './Loader.treat';
+
+const indicators = [...Array(3)];
+
+interface LoaderProps {
+  size?: keyof Theme['typography']['text'];
+}
+
+const indicatorColour = (
+  backgroundContext?: UseBoxStylesProps['background'],
+) => {
+  if (backgroundContext) {
+    const lookup: Partial<
```

Comment:
Should we hoist this out of the function so it's not re-created on every call?

Reply:
Yep, it no longer depends on anything in the scope. Will do